static/lang/re-en.json

```diff
@@ -67,117 +67,117 @@
+                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3d6[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}.",
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*3d6)[persistent,acid]]] and [[/r (3[splash])[acid]]]{3 acid splash damage}."
                 },
                 "Lesser": {
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 1d6[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}",
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*1d6)[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}"
+                    "success": "<strong>Effect</strong> The bomb also deals [[/r 1d6[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}.",
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*1d6)[persistent,acid]]] and [[/r (1[splash])[acid]]]{1 acid splash damage}."
                 },
                 "Major": {
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 4d6[persistent,acid]]] and [[/r (4[splash])[acid]]]{4 acid splash damage}",
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*4d6)[persistent,acid]]] and [[/r (4[splash])[acid]]]{4 acid splash damage}"
+                    "success": "<strong>Effect</strong> The bomb also deals [[/r 4d6[persistent,acid]]] and [[/r (4[splash])[acid]]]{4 acid splash damage}.",
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*4d6)[persistent,acid]]] and [[/r (4[splash])[acid]]]{4 acid splash damage}."
                 },
                 "Moderate": {
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 2d6[persistent,acid]]] and [[/r (2[splash])[acid]]]{2 acid splash damage}",
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*2d6)[persistent,acid]]] and [[/r (2[splash])[acid]]]{2 acid splash damage}"
+                    "success": "<strong>Effect</strong> The bomb also deals [[/r 2d6[persistent,acid]]] and [[/r (2[splash])[acid]]]{2 acid splash damage}.",
+                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r (2*2d6)[persistent,acid]]] and [[/r (2[splash])[acid]]]{2 acid splash damage}."
                 }
             },
             "AlchemistsFire": {
                 "Greater": {
-                    "success": "<strong>Effect</strong> The bomb also deals [[/r 3[persistent,fire]]] and [[/r (3[splash])[fire]]]{3 fire splash damage}",
-                    "criticalSuccess": "<strong>Effect</strong> The bomb also deals [[/r 2*3[persistent,fire]]] and [[/r (3[splash])[fire]]]{3 fire splash damage}"
```

Comment:
This communicates that it's standard crit damage.

Reply:
You mean the 2*3?
tbh, I'm not sure how to handle this correctly. 2*3[persistent,fire] creates an incorrect damage formula, (2*3)[persistent, fire] creates an incorrect button label:

![image](https://user-images.githubusercontent.com/66370966/210725862-a99c7256-6f22-4245-b4f7-d441a5c1358a.png)
![image](https://user-images.githubusercontent.com/66370966/210726031-14a53d72-bcee-4287-a0c0-e2d9163d68a4.png)
![image](https://user-images.githubusercontent.com/66370966/210726075-a314881c-bf04-42c1-b967-d17ed561e826.png)

[[/r (2*3)[persistent,fire]]]{2*3 persistent fire} would do the job, but that's a lot of custom labels.

Reply:
Don't worry about the labels; we'll take care of that: they just need parentheses

Reply:
Ok, changed